### ISSUE TITLE
Style Services/Hours app to differentiate rows

### ISSUE THREAD

**Issue by lobobabysaurus** [lobobabysaurus/NewAppearances#36]
It is possible to differentiate iteration number in a for loop in Django templates.  This should be used to create slightly different colors for each row so that it looks more structured and stylish.  You could have logic that says cast every other row to a different class and apply that style.

----------------------------------------
- Bitbucket: https://bitbucket.org/philsim7/newappearances/issue/15
- Originally reported by: Phil Simmons
- Originally created at: 2015-05-12T19:58:25.858

**Comment by lobobabysaurus** [comment]
As part of this can you add that full service list I gave you for the business as test data for the services app and create a fixture for it.  It will make testing this so much easier and it will be relatively quick and simple and it is way overdue.

----------------------------------------
Original comment by: Phil Simmons

**Comment by lobobabysaurus** [comment]
I'm having trouble finding the copy you made me.  I know, I'm an ass. I'm going to look for it some more, but if I can't find it I'm just going to take pictures of the original this Friday/Saturday and email them to myself.  

On a positive note, the first part of this was really easy.  Either python or django have a built-in for loop counter variable, and I just did a mod 2 check on it each pass to alternate rows

----------------------------------------
Original comment by: Geoffrey Pitman

**Comment by lobobabysaurus** [comment]
Added a PR with the proper fixture data, also that is what I had looked up and was gonna use for differentiating rows. Make sure you put a PR up as soon as you're finished up with the row stuff

----------------------------------------
Original comment by: Phil Simmons

**Comment by lobobabysaurus** [comment]
Grabbed to knock out hours app

----------------------------------------
Original comment by: Phil Simmons

**Comment by lobobabysaurus** [comment]
Added to services app by @gpitm379 pull request #24.  Functions by checking if the django for loop iteration to give a different class to each row.  The rows are large enough in CSS to meet and the alternating classes are given different colors.  They are also shrunk in width a bit to look nicer in context to the background.  I applied the exact same approach to the hours app and refactored the CSS in pull request #25

----------------------------------------
Original comment by: Phil Simmons
